nuxt.config.ts

```diff
@@ -174,12 +174,16 @@ const config: Configuration = {
     dir: 'public',
     routes: (function () {
       const events = fs.readdirSync(path.resolve(__dirname, 'content', 'events'))
-        .filter(filename => path.extname(filename) === '.md')
+        .filter(isMarkdownContent)
         .map(filename => `/events/${path.parse(filename).name}`)
       const experiments = fs.readdirSync(path.resolve(__dirname, 'content', 'experiments'))
-        .filter(filename => path.extname(filename) === '.md')
+        .filter(isMarkdownContent)
         .map(filename => `/experiments/${path.parse(filename).name}`)
       return events.concat(experiments)
+
+      function isMarkdownContent (filename: string): boolean {
```

Comment:
Could you change the function name to something that represent what is doing? It's called `isMarkdownContent` but it check also if it's the `README` file

Reply:
The idea was to highlight the `README` is not content, thus the name. I'll give another take.